src/__tests__/Header.test.js

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { MemoryRouter } from 'react-router-dom';
+import Header from '../components/Header';
+
+describe('test for Header', () => {
+  test('test to return particular string', () => {
```

Comment:
- [ ] I kindly suggest that you make sure that all your tests pass because it is a project requirement.

![image](https://user-images.githubusercontent.com/52934128/175775034-21581273-1e83-4132-b74b-718fc603fad4.png)